common/build/build-common/api-extractor-common.json

```diff
@@ -34,16 +34,6 @@
 		"apiJsonFilePath": "<projectFolder>/_api-extractor-temp/doc-models/<unscopedPackageName>.api.json"
 	},
 
-	/**
-	 * Configures how the .d.ts rollup file will be generated.
-	 */
-	"dtsRollup": {
-		"enabled": false,
```

Comment:
Is the default for this flag false? Just wondering if we need to keep the `dtsRollup` config with just the `"enabled": false` line.